apps/mull-ui/src/app/components/event-card/event-card.tsx

```diff
@@ -34,25 +36,29 @@ export const EventCard = ({ event, style = {}, onClick, isJoined = false }: Even
         <div>{time.replace(/\s/g, '')}</div>
       </div>
       {/* TODO: Add/remove user to event on press */}
-      <button
-        onClick={(e) => {
-          e.stopPropagation();
-          setJoined(!joined);
-          if (joined === false) {
-            joinEvent({ variables: { eventId: event.id } });
-          } else {
-            leaveEvent({ variables: { eventId: event.id } });
-          }
-        }}
-        className={`event-card-join ${joined ? 'joined' : 'not-joined'}`}
-        id={'event-card-join'}
-      >
-        {joined ? (
-          <FontAwesomeIcon icon={faCheck} />
-        ) : (
-          <img src="../../../assets/icons/join-icon.svg" className="join-icon" alt="Join" />
-        )}
-      </button>
+
+      {currentDate.getTime() > new Date(event.endDate).getTime() ? null : (
```

Comment:
⚠ Since you already have this condition in place to check if the event is passed to prevent people from joining, you should also check if the current user is the owner and hide the button if that's also the case.

Might as well do it now to save the trouble.

Reply:
done